plugins/admin/pkg/command/autoscaling/autoscaling.go

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package autoscaling
+
+import (
+	"github.com/spf13/cobra"
+	"knative.dev/client-contrib/plugins/admin/pkg"
+)
+
+// domainCmd represents the domain command
+func NewAutoscalingCmd(p *pkg.AdminParams) *cobra.Command {
+	var AutoscalingCmd = &cobra.Command{
+		Use:   "autoscaling",
+		Short: "Manage autoscaling config",
+		Long: `Manage autoscaling provided by Knative Pod Autoscaler (KPA). For example:
+
+kn admin autoscaling set - to manage autoscaling config`,
```

Comment:
What about also adding the `unset` example too?

Reply:
Sure, need a refine of how to enable/disable scale-to zero and will also add examples here, too